packages/components/src/Banner/Banner.tsx

```diff
@@ -37,9 +44,12 @@ export function Banner({
   dismissible = true,
   icon,
   onDismiss,
+  controlledVisiblity,
 }: BannerProps) {
   const [showBanner, setShowBanner] = useState(true);
   const bannerIcon = icon || getBannerIcon(type);
+  const visible =
+    typeof controlledVisiblity === "undefined" ? true : controlledVisiblity;
```

Comment:
Change this to: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing